src/components/Icons/IconNext.tsx

```diff
@@ -1,6 +1,16 @@
+import { useState, useEffect, useMemo } from 'react';
 import { Icon } from '@aws-amplify/ui-react';
+import { v4 as uuidv4 } from 'uuid';
 
 export const IconNext = ({ ...rest }) => {
+  const [gradient1Id, setGradient1Id] = useState<string>('');
+  const [gradient2Id, setGradient2Id] = useState<string>('');
```

Comment:
Previously, I was setting the ids like this:
```
const gradient1Id = uuidv4();
const gradient2Id = uuidv4();
```
But it was throwing this error in the console. 
<img width="1468" alt="Screenshot 2023-10-24 at 11 06 29 AM" src="https://github.com/aws-amplify/docs/assets/376920/2a53f525-0c25-4874-bb84-c7669c2744db">
Setting them client side (inside a useEffect) works?